README.md

```diff
@@ -6,25 +6,93 @@
+
+## Key Features
+
+Please note: These are the instructions for the latest OSV-Scanner V2 beta. If you are using V1, checkout the V1 [README](https://github.com/google/osv-scanner-v1) and [documentation](https://google.github.io/osv-scanner-v1/) instead. 
+
+### Scanning a source directory
+
+`osv-scanner scan source -r /path/to/your/dir`  
+This command will recursively scan the specified directory for any supported package files, such as `package.json`, `go.mod`, `pom.xml`, etc. and output any discovered vulnerabilities.
+
+For advanced usage, please check out the [documentation](https://google.github.io/osv-scanner/usage).
+
+#### Supported Lockfiles
+
+OSV-Scanner supports 11+ language ecosystems and 19+ lockfile types. To check if your ecosystem is covered, please check out our [detailed documentation](https://google.github.io/osv-scanner/supported-languages-and-lockfiles/). 
+
+### Container Scanning
+
+OSV-Scanner also supports comprehensive, layer-aware scanning for container images to detect vulnerabilities in operating system packages and language-specific dependencies.
+
+`$ osv-scanner scan image my-image-name`
+
+![image](https://github.com/user-attachments/assets/9e3e9c59-1948-45ab-9717-61fcbe3c7cc3)
 
-Read our [detailed documentation](https://google.github.io/osv-scanner) to learn how to use OSV-Scanner.
+### Guided Remediation (experimental)
 
-## Go toolchain compatibility policy
+OSV-Scanner provides guided remediation, a feature that suggests package version upgrades based on criteria such as dependency depth, minimum severity, fix strategy, and return on investment.    
+We currently support remediating vulnerabilities in the following files:
+
+| Ecosystem | File Format (Type) | Supported [Remediation Strategies](#remediation-strategies) |
+| :---- | :---- | :---- |
+| npm | `package-lock.json` (lockfile) | [`in-place`](https://google.github.io/osv-scanner/experimental/guided-remediation/#in-place-lockfile-remediation) |
+| npm | `package.json` (manifest) | [`relock`](https://google.github.io/osv-scanner/experimental/guided-remediation/#in-place-lockfile-remediation) |
+| Maven | `pom.xml` (manifest) | [`override`](https://google.github.io/osv-scanner/experimental/guided-remediation/#override-dependency-versions) |
+
+This is available as a headless CLI command, as well as an interactive mode.
+
+#### Example (for npm)
+
+`$ osv-scanner fix \--max-depth=3 \--min-severity=5 \--ignore-dev  \--non-interactive \--strategy=in-place \-L path/to/package-lock.json`
+
+#### Interactive mode (for npm)
+
+`$ osv-scanner fix -M path/to/package.json -L path/to/package-lock.json `
+
+<img src="https://google.github.io/osv-scanner/images/guided-remediation-relock-patches.png" alt="Screenshot of the interactive relock results screen with some relaxation patches selected">
+
+More information can be found [here](https://google.github.io/osv-scanner/experimental/guided-remediation/).
+
+### Call analysis
```

Comment:
I would remove the Call analysis and C++ sections, and just list them as short bullet points under source scanning, since they go with that part of the scan process only.